server/handlers/client.js

```diff
@@ -118,12 +153,11 @@ exports.createClientHandler = async function (req, res, next) {
     console.error(`CreateClient() -> couldn't retrieve newly created client`);
   }
 
-  return res.json(ClientInfo.fromMongoJSON(result));
+  return res.json(ClientInfoResult.fromMongoJSON(result));
```

Comment:
We just need to return the id, there is no need to fetch the new client

Reply:
I can't implement all the changes related to old sprints APIs. I think it will be a task for the next sprint